### DISCUSSION TITLE
Obfuscation of Tailwind Classes via mangle-css-class-webpack-plugin

### DISCUSSION THREAD

**Discussion by logan-jobzmall** [tailwindlabs/tailwindcss#7239] [Help]
Hi,

I was wondering if anyone in the community knows of a `mangle-css-class-webpack-plugin` configuration that handles Tailwind V3. I know this is a pretty common practice, but I was wondering if this something that someone else was able to get working already with Tailwind V3/JIT compiler. 

I was reading [this article for inspiration](https://medium.com/geekculture/obfuscate-tailwind-classes-in-a-laravel-project-71694b9d8ac1) on this discussion, but I am trying to get something working with Tailwind + Angular.

If there is anything else that can be used to do this that I was not able to to find as well, it would be greatly appreciated.

**Comment by thenaim** [comment]
@logan-jobzmall, Hi! Any news on this? I'm also trying to find a solution.

**Reply by logan-jobzmall** [reply]
Unfortunately, I haven't been successful yet because Angular's underlying mechanisms - but I will keep trying when I have extra time and will report back if I am successful

**Comment by sonofmagic** [comment]
Hi, i write a plugin to obfuscate tailwindcss classes. you can have a try.
npm: [unplugin-tailwindcss-mangle](https://www.npmjs.com/package/unplugin-tailwindcss-mangle)
source code: https://github.com/sonofmagic/tailwindcss-mangle/tree/main/packages/unplugin-tailwindcss-mangle

Now, this plugin support `webpack` and `vite`.it work's when `webpack/vite` build your app.
